migrations/20200826075146_van_survey_questions.js

```diff
@@ -0,0 +1,134 @@
+exports.up = function(knex) {
+  return knex.schema.raw(`
+    create type van_data_collection_status as enum ('active', 'archived', 'inactive');
+
+    alter table public.external_system add constraint external_system_pkey primary key (id);
+
+    create table public.external_survey_question (
+      system_id uuid not null references external_system(id),
+      external_id integer not null,
+      type text,
+      cycle integer,
+      name text,
+      medium_name text,
+      short_name text,
+      script_question text,
+      status van_data_collection_status,
```

Comment:
I think it's fine to keep it VAN specific everywhere except enums, pretty much. PostgreSQL enums are hard to change! We could not check in adding a value to an enum in knex migration, for example, since it can't be run in a transaction. No good!